File: NHSOnline.Backend.Worker/GpSystems/Suppliers/Emis/Models/Appointment.cs
namespace NHSOnline.Backend.Worker.GpSystems.Suppliers.Emis.Models
{
    public class Appointment
    {
        public int SlotId { get; set; }
        public string StartTime { get; set; }
        public string EndTime { get; set; }
        public int SessionId { get; set; }
    }
}

File: bddtests/src/main/kotlin/mocking/emis/models/Appointment.kt
package mocking.emis.models

data class Appointment(
        val slotId: Int,
        val sessionId: Int = 1,
        var startTime: String = "",
        var endTime: String = "",
        var bookingDate: String = "",
        val bookingReason: String = "",
        var BookingMethod: String = "PatientFacingServices",
        var slotTypeName: String = "Default",
        var slotTypeStatus: SlotTypeStatus = SlotTypeStatus.Unknown,
        var vidyoRoomUri: String? = null,
        var telephoneAppointmentDetails: TelephoneAppointmentDetails? = null
)
